package.json

```diff
@@ -9,11 +9,11 @@
   "icon": "images/csharpIcon.png",
   "preview": false,
   "bugs": {
-    "url": "https://github.com/microsoft/vscode-dotnettools/issues"
+    "url": "https://github.com/dotnet/vscode-csharp"
```

Comment:
Was the intention though that the vscode-dotnettools repo is still the primary place to file bugs (and we'll move them around from there?)

Reply:
No - when you report an issue you select an extension to report against.  If someone selects the C# extension, bugs should go here.